src/components/layouts/_base.njk

```diff
@@ -1,11 +1,11 @@
 <!doctype html>
-<html lang="{{ lang }}">
+<html lang="{% if lang %}{{ lang }}{% else %}en{% endif %}">
   <!-- generated by {{ package.name }}@{{ package.version }} -->
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>{{ meta.title }}</title>
+    <title>{% if meta.title %}{{ meta.title }}{% else %}USWDS Fractal example document{% endif %}</title>
```

Comment:
I was tempted to just remove the `{% if %}` statement from both of these modified lines entirely, as I doubt we're actually defining `lang` or `meta.title` anywhere, but I left them in just in case...